src/Microsoft.Framework.ApplicationHost2/Microsoft.Framework.ApplicationHost2.kproj

```diff
@@ -13,5 +13,8 @@
   <PropertyGroup>
     <SchemaVersion>2.0</SchemaVersion>
   </PropertyGroup>
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <ProduceOutputsOnBuild>True</ProduceOutputsOnBuild>
```

Comment:
This might reappear in some future commits on this branch but I'll nuke it before final merging.